src/telliot_feed_examples/sources/blockhash_aggregator.py

```diff
@@ -143,6 +143,9 @@ class TellorRNGManualSource(DataSource[Any]):
 
     timestamp = 0
 
+    def setTimestamp(self, timestamp: int) -> None:
```

Comment:
Small thing, but Python convention is snake case, so it would be `set_timestamp` instead of `setTimestamp`